xc/device.ml

```diff
@@ -1411,24 +1411,20 @@ end
 module Vusb = struct
   let cleanup domid =
     try
-      let stdout, stderr = Forkhelpers.execute_command_get_output Device_common.usb_reset_script ["cleanup"; "-d"; string_of_int domid] in
-      debug "Got %s %s" stdout stderr
+      ignore(Forkhelpers.execute_command_get_output !Xc_resources.usb_reset_script ["cleanup"; "-d"; string_of_int domid])
     with _ ->
       warn "Failed to clean up VM %s" (string_of_int domid)
 
   let call_usb_reset hostbus hostport domid pid action priv =
     try
       match action, priv with
       | "attach", false ->
-        let stdout, stderr = Forkhelpers.execute_command_get_output Device_common.usb_reset_script [ "attach"; hostbus ^ "-" ^ hostport; "-d"; string_of_int domid; "-p"; string_of_int pid] in
-        debug "Got %s %s" stdout stderr
+        ignore(Forkhelpers.execute_command_get_output !Xc_resources.usb_reset_script [ "attach"; hostbus ^ "-" ^ hostport; "-d"; string_of_int domid; "-p"; string_of_int pid])
```

Comment:
The output would be actually useful for debugging. I suggest:

```
let cmd = [ "attach" ; ... ]  in (* please avoid long lines and break this up *)
let stdout, stderr = Forkhelpers.execute_command_get_output Device_common.usb_reset_script cmd in
debug "usb_reset_script '%s' returned stdout='%s' stderr='%s'" (String.concat " " cmd) stdout stderr
```
Since the script is called in several places, I would define a function and pass in just the arguments:

```
let exec_usb_reset_script argv =
  let stdout, stderr = Forkhelpers.execute_command_get_output Device_common.usb_reset_script argv in
  debug "usb_reset_script '%s' returned stdout='%s' stderr='%s'" (String.concat " " argv) stdout stderr
...
exec_usb_reset_script ["attach"; ..]
```

Reply:
Thanks. Updated. Have squashed the refine commit.